experiments/gail_benchmark.csv

```diff
@@ -0,0 +1,32 @@
+env_config,rollout_glob,n_gen_steps_per_epoch,n_epochs,n_demonstrations,

```

Comment:
Consider renaming to `gail_benchmark_config.csv`?

Reply:
Why is there an empty column at the end?

Reply:
The empty column is somehow necessary for `parallel` to parse the `n_demonstrations` column correctly. Without it, the `n_demonstrations` is simply ignored. (I should clarify this in the script's comments)

Reply:
https://stackoverflow.com/questions/57280651/gnu-parallel-doesnt-work-without-trailing-commas-in-csv

Reply:
I see you've removed the empty columns now -- did you fix this? It seems like an odd error. I notice you have trailing spaces at the end of each line, which could potentially cause problems, but is probably OK (`int("4 ") == 4 == int("4")`).

Reply:
Hmm, I don't see trailing spaces in the CSV. 
```
shwangster@airbears2-10-142-35-202:/tmp/parallel$ cat -vet gail_benchmark_config.csv 
env_config,rollout_glob,n_gen_steps_per_epoch,n_epochs,n_demonstrations$
cartpole,CartPole-v1.pkl,5000,300,1$
cartpole,CartPole-v1.pkl,5000,300,4$
cartpole,CartPole-v1.pkl,5000,300,7$
cartpole,CartPole-v1.pkl,5000,300,10$
mountain_car,MountainCar-v0.pkl,5000,300,1$
```

I suppose you mean the actual parallel command.

Reply:
Yeah, it's fixed now! This is the title of one of the commits 
e0d9ebb